xbmc/addons/LanguageResource.cpp

```diff
@@ -92,14 +92,19 @@ bool CLanguageResource::IsInUse() const
   return StringUtils::EqualsNoCase(CServiceBroker::GetSettingsComponent()->GetSettings()->GetString(CSettings::SETTING_LOCALE_LANGUAGE), ID());
 }
 
-void CLanguageResource::OnPostInstall(bool update, bool modal)
+void CLanguageResource::OnPostInstall(bool update, bool modal, int optionalTypeDependend)
 {
   if (!g_SkinInfo)
     return;
 
-  if (IsInUse() ||
-     (!update && !modal &&
-       (HELPERS::ShowYesNoDialogText(CVariant{Name()}, CVariant{24132}) == DialogResponse::YES)))
+  if (optionalTypeDependend == POST_INSTALL_LANGUAGE_RESOURCE_NO_SELECT)
+  {
+    // Force to reload current strings
+    g_langInfo.SetLanguage("", true);
```

Comment:
maybe this can be defined as a new method in `g_langInfo`? Without the comment it's hard to understand the reason for this call (or what its intended behavior is)

Reply:
Has taken from:
https://github.com/xbmc/xbmc/blob/master/xbmc/Application.cpp#L4677-L4679

Maybe I can add a function above the other as?:
```cpp
void CLangInfo::ReloadCurrentLanguage()
{
  SetLanguage("", true);
}

bool CLangInfo::SetLanguage(std::string language /* = "" */, bool reloadServices /* = true */)
{
...
```

Only not sure if it makes really sense if only called by one place.